runtime/compiler/x/codegen/J9AheadOfTimeCompile.cpp

```diff
@@ -294,6 +294,7 @@ uint8_t *J9::X86::AheadOfTimeCompile::initializeAOTRelocationHeader(TR::Iterated
       case TR_ProfiledMethodGuardRelocation :
          {
          guard = (TR_VirtualGuard *)relocation->getTargetAddress2();
+         TR_OpaqueClassBlock *inlinedCodeClass = guard->getThisClass();
```

Comment:
This is the profiled class, yes? Not the defining class of the method, which may have been inherited

Reply:
Yeah that's right, we don't store the romclass of the defining class of the method.